lib/entry-index.js

```diff
@@ -266,9 +256,8 @@ function _bucketEntries (data, filter) {
     let obj
     try {
       obj = JSON.parse(pieces[1])
-    } catch (e) {
-      // Entry is corrupted!
-      return
+    } catch (_) {
+      // eslint-ignore-next-line no-empty-block
```

Comment:
github diff makes it harder to see. was the return removed here cause it's the end of the function and returns anyway?

Reply:
the return was removed because immediately after the catch we had a condition where we only push the result into the entries array if it's truthy, so the catch calling `return` is pointless